source/class/qx/ui/basic/Image.js

```diff
@@ -936,11 +936,6 @@ qx.Class.define("qx.ui.basic.Image",
 
       if (isFont) {
         var sparts = source.split("/");
```

Comment:
Is sparts still needed?

Reply:
Yes, in line  [L945](https://github.com/adrelino/qooxdoo/blob/73c9bf4879e00a7804a881f0a6c5c8362c7be1ff/source/class/qx/ui/basic/Image.js#L954):
```js
var size = parseInt(sparts[2] || qx.theme.manager.Font.getInstance().resolve(source.match(/@([^/]+)/)[1]).getSize());
```
for the sizing of the image. 

I know that this line is duplicated now with my new code in ResourceManager, but this was for the sake of a more minimal diff. Only Image.js considers the sizing hint from the material icon, Decorator.js doesn't.

Reply:
Ah! I see! OK. Thank you.